### ISSUE TITLE
Karma tests fail because of missing shadow$loader

### ISSUE THREAD

**Issue by macluck** [thheller/shadow-cljs#351]
Hey,

I'm having a problem with running ci tests with karna in latest version of shadow-cljs (2.4.24).
Config for ci tests is:

```edn
:ci-test  {:target     :karma
           :output-to  "./target/ci.js"}
```

Karma tests fail with: `Uncaught ReferenceError: shadow$loader is not defined`

I suspect that it got broken in https://github.com/thheller/shadow-cljs/commit/8b78485e02dfc8db3e5852e558421d02298d1371 , because there were no problems prior this commit.

**Comment by thheller** [comment]
Hmm do you use `shadow.loader` in any way in your tests? Do you test anything that would actually try to load any modules dynamically?

Right now the error can occur just by loading `shadow.loader` which of course it not idea. I can fix that part easily but I wonder if the tests would actually rely on modules being present?

**Comment by thheller** [comment]
The issue should be fixed `2.4.25`.

**Comment by macluck** [comment]
Thank you, it is indeed fixed now.
